sonar-helm-for-iac/build.gradle.kts

```diff
@@ -113,6 +113,8 @@ if (!isCi) {
         // It is too difficult to check if image is built; Docker takes care of it anyway.
         setErrorOutput(System.out)
 
+        val isCiBuild = "ci" == System.getProperty("buildEnv")
```

Comment:
We already have an env variable `CI` on Cirrus (and `isCi` variable above in this script). Maybe it can be reused here?